tiatoolbox/tools/registration/__init__.py

```diff
@@ -0,0 +1 @@
+"""Models package for the models implemented in tiatoolbox."""
```

Comment:
```suggestion
"""Models package for the models implemented in tiatoolbox."""
from tiatoolbox.tools.registration import wsi_registration
```

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/372/commits/b11fde44306914341e2853647df36cb8dadbd0fc